packages/realm/src/assert.ts

```diff
@@ -138,17 +141,6 @@ assert.never = (value: never, target?: string): never => {
 
 // SDK specific
 
-assert.primaryKey = (value: unknown, target?: string): asserts value is PrimaryKey => {
```

Comment:
Is this no longer needed?

Reply:
I simplified the only calling code to no longer need it.